src/filters/math.rs

```diff
@@ -247,32 +247,32 @@ mod tests {
     #[test]
     fn unit_modulo() {
         assert_eq!(
-            unit!(modulo, Value::scalar(3_f32), &[Value::scalar(2_f32)]),
-            Value::scalar(1_f32)
+            unit!(modulo, Value::scalar(3_f64), &[Value::scalar(2_f64)]),
+            Value::scalar(1_f64)
         );
         assert_eq!(
-            unit!(modulo, Value::scalar(3_f32), &[Value::scalar(3.0)]),
-            Value::scalar(0_f32)
+            unit!(modulo, Value::scalar(3_f64), &[Value::scalar(3.0)]),
+            Value::scalar(0_f64)
         );
         assert_eq!(
-            unit!(modulo, Value::scalar(24_f32), &[Value::scalar(7_f32)]),
-            Value::scalar(3_f32)
+            unit!(modulo, Value::scalar(24_f64), &[Value::scalar(7_f64)]),
+            Value::scalar(3_f64)
         );
         assert_eq!(
-            unit!(modulo, Value::scalar(183.357), &[Value::scalar(12_f32)]),
+            unit!(modulo, Value::scalar(183.357), &[Value::scalar(12_f64)]),
             Value::scalar(3.3569946)
```

Comment:
> Not sure what's causing the failing float test - any ideas?

I'm assuming we were rounding due to f32 and now with f64, we have more precision.  IUn theory, we just change the result we are comparing aginst.